lib/Backends/CPU/CPUBackend.cpp

```diff
@@ -455,3 +491,19 @@ llvm::StringRef CPUBackend::getLibjitBitcode() const {
   return llvm::StringRef(reinterpret_cast<const char *>(libjit_bc),
                          libjit_bc_size);
 }
+
+/// Returns true if network supports Type Lowering from T1 to T2.
+/// Allows the backend to control node white/black list for optimization.
```

Comment:
This is not clear -- this is returning a blacklist of ops that cannot be converted, right? I'd just say that specifically, otherwise it's not clear how this is used. Same for all places where this comment exists.